src/program.js

```diff
@@ -191,7 +194,8 @@ export class Program {
       }
 
       configFiles.forEach((configFileName) => {
-        const configObject = loadJSConfigFile(configFileName);
+        const configObject = configFileName.endsWith('package.json') ?
```

Comment:
Since you don't need a custom loader you could use this opportunity to look for the `webExt` key:

```js
let configObject = loadJSConfigFile(configFileName);
if (configFileName.endsWith('package.json')) {
  log.debug('Looking for webExt key in package.json file');
  configObject = configObject.webExt || {};
}
```